### ISSUE TITLE
Extensions default to the same colour

### ISSUE THREAD

**Issue by vincentbriglia** [LLK/scratch-vm#1151]
### Expected Behavior

Specifying primary, secondary and tertiary colours on an extension should allow the extension to use those colours instead of the default hardcoded colours

### Actual Behavior

All extensions have the same colour scheme, without the ability to change it 

### Steps to Reproduce

Add multiple extensions

### Operating System and Browser

Mac OS 10.13.3, Chrome Version 66.0.3359.139 (Official Build) (64-bit)

**Comment by regcole** [comment]
# WorkAround
**Step 1:** In info section of your extension, add the following JSON fields to set colors
```js
 getInfo () {
        return {
            id: 'someId',
            name: 'sampleName',
            menuIconURI: menuIconURI,
            categoryColor: '#38913f',
            secondCategoryColor: '#38913f',
            ........
```

**Step 2:** 
Then edit the `_registerExtensionPrimitives` method in `runtime.js` to use those colors instead of the default ones like so:
```javascript
/**
     * Register the primitives provided by an extension.
     * @param {ExtensionMetadata} extensionInfo - information about the extension (id, blocks, etc.)
     * @private
     */
    _registerExtensionPrimitives (extensionInfo) {
       // use default colors if no color is specified 
        let categoryColor = "#FF6680";
        let secondCategoryColor = "#FF4D6A";
        let thirdCategoryColor = "#FF4D6A";
        if(extensionInfo.categoryColor) {
            categoryColor = extensionInfo.categoryColor;
        }
        if(extensionInfo.secondCategoryColor) {
            secondCategoryColor = extensionInfo.secondCategoryColor;
        }
        if(extensionInfo.thirdCategoryColor) {
            thirdCategoryColor = extensionInfo.thirdCategoryColor;
        }
        const categoryInfo = {
            id: extensionInfo.id,
            name: maybeFormatMessage(extensionInfo.name),
            blockIconURI: extensionInfo.blockIconURI,
            menuIconURI: extensionInfo.menuIconURI,
            color1: categoryColor,
            color2: secondCategoryColor,
            color3: thirdCategoryColor,
            blocks: [],
            menus: []
        };
```

**Comment by towerofnix** [comment]
Should this be closed now, since #1153 was merged?
